cvat/apps/engine/views.py

```diff
@@ -1148,6 +1158,77 @@ def data(self, request, pk):
         return data_getter(request, db_job.segment.start_frame,
             db_job.segment.stop_frame, db_job.segment.task.data)
 
+    @extend_schema(summary='Method provides a meta information about media files which are related with the job',
+        responses={
+            '200': DataMetaSerializer,
+        }, tags=['tasks'], versions=['2.0'])
```

Comment:
@Marishka17 , could you please look at the definition? I remember that in your patch all tags and versions were removed.

Reply:
Yes, it should be removed

Reply:
Fixed